Sources/Request/Request/RequestBuilder.swift

```diff
@@ -10,35 +10,26 @@ import Foundation
 @_functionBuilder
 public struct RequestBuilder {
     public static func buildBlock(_ params: RequestParam...) -> RequestParam {
```

Comment:
I think that `unzip` and even `CombinedParams` will be unnecessary if we return an array of `RequestParam`s from the builder:
```suggestion
    public static func buildBlock(_ params: RequestParam...) -> [RequestParam] {
        params
    }
```